gateway/src/apicast/cli/template.lua

```diff
@@ -94,7 +94,7 @@ function _M:interpret(str)
     local context = self.context
     local filesystem = self.filesystem
     local filter_set = FilterSet:new()
-    local resource_limit = ResourceLimit:new(nil, 1000, nil)
+    local resource_limit = ResourceLimit:new(1000000, 1000, nil)
```

Comment:
Would be good to have a link to the issue here so we remember why this value is not the default one.